packages/sentry-client-web/src/components/MessageBox.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from "react";
+
+interface IMessageBoxProps {
+    message: string;
+    style?: React.CSSProperties;
+    textStyle?: React.CSSProperties;
+}
+
+export class MessageBox extends React.Component<IMessageBoxProps, any> {
+    public getStyle(): React.CSSProperties {
+        return {
+            display: "flex",
+            alignItems: "center",
+            justifyContent: "center",
+            ...this.props.style
```

Comment:
The flexible `style` interface is cool but definitely not needed. It clutters up `List`